c++/CMakeLists.txt

```diff
@@ -82,11 +82,40 @@ else()
 endif()
 
 # Source =======================================================================
-
+include(CapnProtoMacros)
 add_subdirectory(src)
 
 # Install ======================================================================
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake"
+  VERSION ${VERSION}
+  COMPATIBILITY AnyNewerVersion
```

Comment:
`AnyNewerVersion` should be `ExactVersion`. AFAIK, there aren't any backwards-compatibility guarantees for the API.